src/pages/versions.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
```

Comment:
Could you convert this page to `tsx` (rework to use TypeScript instead of pure JS)? This is how it should look like: https://github.com/facebook/docusaurus/blob/67c40069d1062b5aae530b696e6f66bf9618a696/website/src/pages/versions.tsx#L8